obspy/io/nordic/core.py

```diff
@@ -1390,7 +1390,10 @@ def nordpick(event, high_accuracy=True):
                 warnings.warn("Multiple arrivals for pick - only writing one")
             arrival = arrival[0]
             # Extract weight - should be stored as 0-4, or 9 for seisan.
-            weight = _str_conv(int(arrival.time_weight or 0))
+            if arrival.time_weight is None:
+                weight = ' '
+            else:
+                weight = _str_conv(int(arrival.time_weight or 0))
```

Comment:
since you are handling `None` value explicitely, the `or 0` should not be needed anymore, looks like?

Reply:
That's true.  I changed the code.